packages/ui/react-ui-editor/src/extensions/markdown/highlight.ts

```diff
@@ -155,12 +144,6 @@ export const markdownHighlightStyle = (readonly?: boolean) => {
       { tag: tags.strong, class: bold },
       { tag: tags.strikethrough, class: strikethrough },
 
-      // Naked URLs.
```

Comment:
so this means it will no longer specially format raw urls?

Reply:
Ah good point I forgot about that. It was interfering with URLs inside the text part of markdown links. Should we try to reinstate that functionality? I could potentially work on it as a follow-up issue?

Reply:
yes we just need i decorator (not just CSS) that recognizes links.